index.js

```diff
@@ -310,6 +327,10 @@ const ContainerService = {
   },
 
   sortTabs() {
+    sendEvent({
```

Comment:
I'll try this, but I'm not sure an extra `_sendTelemetryData` utility method will give us any better code re-use than a `this._sendEvent` method gives us.